egg-fabric-on-demand.json

```diff
@@ -0,0 +1,167 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-05-04T22:32:53+02:00",
+    "name": "Fabric - MSH",
+    "author": "JackGlobetrotter",
```

Comment:
Needs to be a valid E-Mail.
`Could not save Pterodactyl\Models\Egg[]: failed to validate data: {"author":["The author must be a valid email address."]}`